R/ncdf.R

```diff
@@ -115,11 +115,7 @@ read_ncdf = function(.x, ..., var = NULL, ncsub = NULL, curvilinear = character(
   if(ncol(all_coord_var) == 0) all_coord_var <- data.frame(variable = NA, X = NA, Y = NA,
                                                            Z = NA, T = NA, bounds = NA)
 
-  if(is.null(proxy_dimensions)) {
-	  nc_prj <- .get_nc_projection(meta$attribute, rep_var, all_coord_var)
-  }
```

Comment:
I've moved this down after I know what coordinate variables (and units) will be needed.